server/fleet/software.go

```diff
@@ -57,9 +57,6 @@ type Software struct {
 	// GenerateCPE is the CPE23 string that corresponds to the current software
 	GenerateCPE string `json:"generated_cpe" db:"generated_cpe"`
 
-	// GeneratedCPEID is the ID of the matched CPE
-	GeneratedCPEID uint `json:"-" db:"generated_cpe_id"`
```

Comment:
This is a breaking API change. Do we need to put it into the next API version?

Reply:
@lukeheath

Reply:
It's not currently used anywhere in the frontend or documented in our API REST documentation.

Reply:
Since this property is not used on the frontend and is not documented in our API docs, I think it's safe to go ahead and remove it now. Please include the change in the changelog so it makes it into the release notes. Thanks for checking!

Reply:
This should be fine, that prop was never exposed through the API (see `json:"-"`)

Reply:
Oh, I didn't see that.